shared/common-adapters/oriented-image.js.flow

```diff
@@ -2,4 +2,6 @@
 import * as React from 'react'
 import type {Props} from './oriented-image.types'
 
-export default class OrientedImage extends React.Component<Props> {}
+export default class OrientedImage extends React.Component<Props> {
+  _context: ?CanvasRenderingContext2D
```

Comment:
is this needed?

Reply:
I was having Flow issues when calling methods on the context object. In theory flow should have inferred the type because it knows the return value of `canvas.getContext()` however it didn't seem to catch on.